main.go

```diff
@@ -89,10 +93,14 @@ func main() {
 	}
 
 	if err = (&controllers.ManagedOCSReconciler{
-		Client:               mgr.GetClient(),
-		Log:                  ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
-		Scheme:               mgr.GetScheme(),
-		AddonParamSecretName: fmt.Sprintf("addon-%v-parameters", envVars[addonNameEnvVarName]),
+		Client:                  mgr.GetClient(),
+		UnrestrictedClient:      getUnrestrictedClient(),
+		Log:                     ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
+		Scheme:                  mgr.GetScheme(),
+		AddonParamSecretName:    fmt.Sprintf("addon-%v-parameters", envVars[addonNameEnvVarName]),
+		DeleteConfigMapName:     envVars[addonNameEnvVarName],
+		DeleteConfigMapLabelKey: fmt.Sprintf("api.openshift.com/addon-%v-delete", envVars[addonNameEnvVarName]),
+		AddonSubscriptionName:   fmt.Sprintf("addon-%v", envVars[addonNameEnvVarName]),
```

Comment:
We have 4 instances of `envVars[addonNameEnvVarName]` in this code section. Can we please refactor into local var ```namespace := envVars[addonNameEnvVarName]``` just before line 95, and use it instead?